src/react-imgix.jsx

```diff
@@ -367,9 +368,13 @@ class SourceImpl extends Component {
 }
 SourceImpl.displayName = "ReactImgixSource";
 
-const ReactImgixWrapped = compose(ShouldComponentUpdateHOC)(ReactImgix);
-const Picture = compose(ShouldComponentUpdateHOC)(PictureImpl);
-const Source = compose(ShouldComponentUpdateHOC)(SourceImpl);
+let ReactImgixWrapped = compose()(ShouldComponentUpdateHOC)(ReactImgix);
+let Picture = compose()(ShouldComponentUpdateHOC)(PictureImpl);
+let Source = compose()(ShouldComponentUpdateHOC)(SourceImpl);
+
+ReactImgixWrapped = mergeComponentPropsHOF(processPropsHOF(ReactImgix))
+Picture = mergeComponentPropsHOF(processPropsHOF(PictureImpl))
+Source = mergeComponentPropsHOF(processPropsHOF(SourceImpl))
```

Comment:
Would maybe be a little better if these were merged together?

Reply:
💯 , I was running into issues having this all happen on one line but I'll give it another try and hopefully it was just a one-time-quirk.